changelog/january2025/2025-01-13-jobs-added-serverless-jobs-console-improvements.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Serverless Jobs console improvements
+status: added
+author:
+    fullname: 'Join the #serverless-jobs channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2025-01-13
+category: serverless
+product: jobs
+---
+
+- Added easy access to unique identifier of ressources in Job Definition view
+- Added direct access to documentation in listing of Job Definitions
```

Comment:
@SamyOubouaziz can you confirm this is right? Looking at the console this is what it seems to be to me